data/campaigns/The_Hammer_of_Thursagan/scenarios/10_The_Siege_of_Kal_Kartha.cfg

```diff
@@ -223,7 +223,7 @@
                 condition=lose
             [/objective]
             [objective]
-                description= _ "Death of West Gate commander"
+                description= _ "Death of Dulcatulos"
```

Comment:
Was there any particular reason to refer to him as the "west gate commander"? I'm just thinking that you could have both, if it seems like a good idea - "Death of Dulcatulos, the West Gate commander", or similar. But no idea if that's a good idea.